package.json

```diff
@@ -91,6 +91,7 @@
     "jest": "^28.1.3",
     "lerna": "^5.2.0",
     "lint-staged": "^13.0.1",
+    "node-gyp": "^9.0.0",
```

Comment:
Why is this a new dependency? :thinking:

Reply:
I added it as without it the command `yarn local-network:up --build` was failing to install `ssh` as it needs `node-gyp` but is not in its dependencies.
This is strange as it seems this is happening only on my local machine...